### ISSUE TITLE
How to configure pom.xml

### ISSUE THREAD

**Issue by keshwans** [kawasima/handlebars-maven-plugin#7]
How to we setup the plugin in pom.xml ?

mvn package with the below plugin setup gives me errors:

[ERROR] Failed to execute goal net.unit8.maven.plugins:handlebars-maven-plugin:0
.2.0:precompile (handlebars-precompile) on project myProj2: Failure fetch handlebar
s. MalformedURLException: NullPointerException -> [Help 1]

```
        <plugin>
            <groupId>net.unit8.maven.plugins</groupId>
            <artifactId>handlebars-maven-plugin</artifactId>
            <version>0.2.0</version>
            <executions>
                <execution>
                    <id>handlebars-precompile</id>
                    <phase>prepare-package</phase>
                    <goals>
                        <goal>precompile</goal>
                    </goals>
                    <configuration>
                        <sourceDirectory>${project.basedir}/src/main/webapp/myProj/templates</sourceDirectory>
                        <outputDirectory>${project.basedir}/src/main/webapp/myProj/js</outputDirectory>
                        <encoding>UTF-8</encoding>
                        <templateExtensions>handlebars</templateExtensions>
                        <handlebarsName>templates.js</handlebarsName>
                    </configuration>
                </execution>
            </executions>
        </plugin>
```

**Comment by kawasima** [comment]
The handlebarsName parameter means the handlebars.js script itself. If you want to use a different version, this parameter is used. (default version is 1.0.0-rc.1)

If you don't need to specify the version, no handlebarsName parameter is needed.

**Comment by keshwans** [comment]
Since templateExtensions is an array in the plugin, I revised my pom.xml tp specify:

< templateExtensions >
     < templateExtension >handlebars< /templateExtension >
</templateExtensions>

fixed the problem for me.

Also, removed < handlebarsName >templates.js< /handlebarsName >. templates.js was the output file I needed to create. That is th the default output filename.

As pointed out by kawasima, < handlebarsName > is used to for specifying the handlebars compiler. I am using the default one pulled by the plugin.

Thanks for your help
